lib/Transport.cpp

```diff
@@ -104,6 +107,11 @@ void Transport::AcceptConnections()
 
 void Transport::QueueNotification(Message&& msg)
 {
+    if (!m_guestWasPresent)
```

Comment:
Before this PR, the host would only send notification to the guest when the guest was connected (meaning that the guest is present). With scan result notifications, this is no longer the case: whenever the host scans, a notification will be sent. If the guest is absent, the connection will fail after trying to connect for a while.
This code is there to at least avoid this situation until the guest made a first request, at which point it can be assume it will be listening.

Reply:
That makes sense, thanks for the explanation.